src/Lykke.Job.BitcoinPrivateWalletsCache.Core/Domain/BalanceSummaryModel.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Newtonsoft.Json;
+
+namespace Lykke.Job.BitcoinPrivateWalletsCache.Core.Domain
+{
+    public class BalanceSummaryModel
```

Comment:
You can use QBitNinja.Client nuget package

Reply:
it's does not have sence, it's temporary service.